File: Easy/Java/1725. Number Of Rectangles That Can Form The Largest Square.java
class Solution {
    public int countGoodRectangles(int[][] rectangles) {
        int len = 0;
        int count = 0;

        for(int[] rect : rectangles){
            len = Math.max(len, Math.min(rect[0], rect[1]));
        }

        for(int[] rect : rectangles){
            if(Math.min(rect[0], rect[1]) == len)
                count++;
        }

        return count;
    }
}

File: Easy/JavaScript/1725. Number Of Rectangles That Can Form The Largest Square.js
var countGoodRectangles = function(rectangles) {
    let len = 0;
    let count = 0;

    rectangles.forEach(rect =>{
        len = Math.max(len, Math.min(rect[0], rect[1]));
    })

    rectangles.forEach(rect =>{
        if(Math.min(rect[0], rect[1]) == len)
            count++;
    })

    return count;
};
